tailwind.config.js

```diff
@@ -1,8 +1,16 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  darkMode: "class",
   content: ["./content/**/*.{md,html}", "./layouts/**/*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: "var(--global-text-color)",
+        secondary: "var(--global-text-color-light)",
+        background: "var(--global-bg-color)",
+        theme: "var(--global-theme-color)",
```

Comment:
Are these colors the only ones that we use? 

I am thinking of adding some predefined color palettes. I wanted to know your idea.

Reply:
At this moment, this are the only colors we use, more can be added if necessary.

I do not have any idea how multiple pallets are managed, but seems interesting.